classes/reviewForm/ReviewFormDAO.inc.php

```diff
@@ -278,17 +278,7 @@ function &getActiveByAssocId($assocType, $assocId, $rangeInfo = null) {
 		$result =& $this->retrieveRange(
 			'SELECT	rf.*
 			FROM	review_forms rf
-				LEFT JOIN review_assignments rac ON (
-					rac.review_form_id = rf.review_form_id AND
-					rac.date_confirmed IS NOT NULL
-				)
-				LEFT JOIN review_assignments rai ON (
-					rai.review_form_id = rf.review_form_id AND
-					rai.date_notified IS NOT NULL AND
-					rai.date_confirmed IS NULL
-				)
 			WHERE	rf.assoc_type = ? AND assoc_id = ? AND rf.is_active = 1
-			GROUP BY rf.assoc_type, rf.assoc_id, rf.review_form_id, rf.seq, rf.is_active
```

Comment:
No, this removal is intentional.  See https://github.com/pkp/pkp-lib/issues/935#issuecomment-160763433

All these joins do is slow the query!  :-)